### DISCUSSION TITLE
Two player on one device?

### DISCUSSION THREAD

**Discussion by t824601** [flyinghead/flycast#1327] [Q&A]
I am currently running an arcade cab with a keyboard controllers. This device is used to map 4 player controls using keyboard keys.

I have not been able to see how I can get Flycast to configure those 4 distinct joysticks using the single keyboard hardware address.

Is there no way to map 2+ players on one single keyboard? Have been able to do this on Retroarch core version of Flycast, but it seems impossinle under the standalone version.

Originally was looking to set up to lightguns, these two work perfectly, but I also need to set up the start button and coin button for player 1 and 2 on the keyboard so my buttons can be used on the cab.

Anyway to do this, any help is appreciated, thank you!

**Comment by flyinghead** [comment]
See this discussion: #1158

**Reply by t824601** [reply]
Thank you, this helped a ton :)!
